prestashop1.6/upgrade/install-1.2.0.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * tawk.to
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to support@tawk.to so we can send you a copy immediately.
+ *
+ * @author tawkto support@tawk.to
+ * @copyright Copyright (c) 2014-2021 tawk.to
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+function upgrade_module_1_2_0()
+{
+    $sql = "UPDATE "._DB_PREFIX_.bqSQL(Configuration::$definition['table'])." conf
+            SET conf.name = REPLACE(conf.name, '_1', '')
+            WHERE conf.name LIKE 'TAWKTO_WIDGET_%'";
```

Comment:
I think this will cause problems if the database records contains data like
- `TAWKTO_WIDGET_PAGE_ID_2`
- `TAWKTO_WIDGET_PAGE_ID_3`
- etc.